### PR TITLE
Checking softInputMethod and throwing error if set to unsupported mode

### PR DESCRIPTION
**Pull request by anoop44**
#19

### CODE REVIEW

File: keyboardvisibilityevent/src/main/java/net/yslibrary/android/keyboardvisibilityevent/KeyboardVisibilityEvent.java [hunk 1]
```diff
@@ -49,6 +50,12 @@ public static Unregistrar registerEventListener(final Activity activity,
             throw new NullPointerException("Parameter:activity must not be null");
         }
 
+        int softInputMethod = activity.getWindow().getAttributes().softInputMode;
+        if(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE != softInputMethod &&
```
Comment (yshrsmz) [inline keyboardvisibilityevent/src/main/java/net/yslibrary/android/keyboardvisibilityevent/KeyboardVisibilityEvent.java]: seems that sofInputMode state is managed like [this](https://github.com/android/platform_frameworks_base/blob/master/core/java/android/view/WindowManager.java#L1400), so following code would be better

`(activity.getWindow().getAttributes().softInputMode & WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE) != 0`

**Comment by anoop44** [comment]
But wouldn't it ignore the `WindowManager.LayoutParams. SOFT_INPUT_ADJUST_UNSPECIFIED `. It will also act as `WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE` at times

**Comment by yshrsmz** [comment]
you are right. thanks.

**Comment by yshrsmz** [comment]
:+1:

**Pull request merged**